cloud-gcp/src/main/java/com/sequenceiq/cloudbreak/cloud/gcp/view/GcpLoadBalancerMetadataView.java

```diff
@@ -32,7 +32,15 @@ public static String getInstanceGroupParam(int port) {
         return INSTANCE_GROUP_PREFIX + port;
```

Comment:
Instead of having two implementations for these functions (simple as I know the implementations are), could you do something like
```
return getInstanceGroupParam(String.valueOf(port));
```